Dockerfile

```diff
@@ -20,4 +20,4 @@ COPY . ./
 RUN make go-generate
 
 RUN go mod download
-ENTRYPOINT ["go"]
+ENTRYPOINT ["go"]
```

Comment:
💡 Only new line is the change here? :) (Just checking is this preferred intendatation approach or jsut editor doing it)

Reply:
The newline is the result of my editor automatically applying it whenever I save a file. The reason I had to resave the Dockerfile was because mine somehow got changed across my commits and I had to copy & paste the one in main over.